templates/trusted-node/cdk-node-config.toml

```
@@ -69,3 +69,19 @@ Outputs = ["stderr"]
 [AggSender]
 CertificateSendInterval = "1m"
 CheckSettledInterval = "5s"
+
+[ClaimSponsor]
+DBPath = "{{.zkevm_path_rw_data}}claimsponsor.sqlite"
+Enabled = true
+SenderAddr = "{{.zkevm_l2_claimtxmanager_address}}"
+BridgeAddrL2 = "{{.zkevm_bridge_l2_address}}"
+
+[ClaimSponsor.EthTxManager]
+PrivateKeys = [
+    {Path = "/etc/cdk/claimtxmanager.keystore", Password = "{{.zkevm_l2_keystore_password}}"},
+]
+StoragePath = "{{.zkevm_path_rw_data}}ethtxmanager-claimsponsor.sqlite"
+
+[ClaimSponsor.EthTxManager.Etherman]
+URL = "http://{{.l2_rpc_name}}{{.deployment_suffix}}:{{.zkevm_rpc_http_port}}"
+L1ChainID = "{{.zkevm_rollup_chain_id}}"
```

Nitpick, missing new line
```suggestion
L1ChainID = "{{.zkevm_rollup_chain_id}}"

```